### DISCUSSION TITLE
Explorer API - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#7381] [Docs]
# Explorer API - Ultralytics YOLOv8 Docs

Explore and analyze CV datasets with Ultralytics Explorer API, offering SQL, vector similarity, and semantic searches for efficient dataset insights.

https://docs.ultralytics.com/datasets/explorer/api/

**Comment by olmezemre** [comment]
pip install ultralytics[explorer] gives this error:
WARNING: Ignoring invalid distribution -rotobuf (c:\users\emre.olmez\anaconda3\lib\site-packages)
WARNING: ultralytics 8.0.231 does not provide the extra 'explorer'
WARNING: Ignoring invalid distribution -rotobuf (c:\users\emre.olmez\anaconda3\lib\site-packages)
WARNING: Ignoring invalid distribution -rotobuf (c:\users\emre.olmez\anaconda3\lib\site-packages)
WARNING: Ignoring invalid distribution -rotobuf (c:\users\emre.olmez\anaconda3\lib\site-packages)
WARNING: Ignoring invalid distribution -rotobuf (c:\users\emre.olmez\anaconda3\lib\site-packages)

**Reply by olmezemre** [reply]
upgraded to ultralytics 8.0.236 and solved

**Reply by RizwanMunawar** [reply]
@olmezemre, we've launched `ultralytics==8.0.237` in the hope that it addresses this issue.

Thanks
Ultralytics Team!
